index.js

```diff
@@ -76,7 +76,7 @@ function normalizeGPLIdentifiers (argument) {
 }
 
 function endsWith (string, substring) {
-  return string.indexOf(substring) === string.length - 1
+  return string.indexOf(substring) === string.length - substring.length
```

Comment:
So this functionality was broken due to a logic error here?

Reply:
Right.

Reply:
Thanks so much for catching and correcting!

Reply:
🙏 . Thanks for the quick review!